src/app/ipv/middleware.js

```diff
@@ -358,75 +341,54 @@ module.exports = {
       delete req.session.currentPageStatusCode;
     }
   },
+
+  // make considerations for handleEscapeAction pages with no req.body?.journey that defaults to `end`
   handleJourneyAction: async (req, res, next) => {
+    const currentPage = req.params.pageId;
+    const pagesUsingSessionId = [
+      "pyi-suggest-other-options",
+      "pyi-cri-escape",
+      "pyi-kbv-escape-m2b",
+      "pyi-escape-m2b",
+      "page-multiple-doc-check",
+    ];
+    const journeyActions = {
+      end: "end",
+      "address-current": "address-current",
+      "attempt-recovery": "attempt-recovery",
+      "build-client-oauth-response": "build-client-oauth-response",
+      next: "next",
+      "next/end": "end",
+      "next/f2f": "f2f",
+      "next/dcmaw": "dcmaw",
+      "next/bank-account": "bankAccount",
+      "next/passport": "ukPassport",
+      "next/driving-licence": "drivingLicence",
+    };
```

Comment:
Probably best taking this out of here into a separate file probably in the shared folder. Leaving it here in the meantime until I get feedback on the new implementation